### PR TITLE
add function character card and export

### PR DESCRIPTION
**Pull request by Leon910**

### CODE REVIEW

File: components/CharacterCard/CharacterCard.js [hunk 1]
```diff
@@ -1 +1,28 @@
-export function CharacterCard() {}
+export function CharacterCard() {
+  const card = document.createElement("li");
+
+  card.classList.add("card");
+  card.innerHTML = `<div class="card__image-container">
+            <img
+              class="card__image"
```
Comment (E-LeeSchaffer) [inline components/CharacterCard/CharacterCard.js:7]: src and alt should be dynamic

File: components/CharacterCard/CharacterCard.js [hunk 2]
```diff
@@ -1 +1,28 @@
...
+          </div>
+          <div class="card__content">
+            <h2 class="card__title">Rick Sanchez</h2>
+            <dl class="card__info">
+              <dt class="card__info-title">Status</dt>
+              <dd class="card__info-description">Alive</dd>
+              <dt class="card__info-title">Type</dt>
+              <dd class="card__info-description"></dd>
+              <dt class="card__info-title">Occurrences</dt>
+              <dd class="card__info-description">51</dd>
```
Comment (E-LeeSchaffer) [inline components/CharacterCard/CharacterCard.js]: name, status, type, occurences should be dynamic

File: components/CharacterCard/CharacterCard.js (hunk 1 above)
Comment (Leon910) [reply components/CharacterCard/CharacterCard.js:7]: @E-LeeSchaffer we add all the dynamic parts, check again please

**Review by E-LeeSchaffer — COMMENTED** [review]
looks good!

**Pull request merged**